### DISCUSSION TITLE
Showing pull request author / review request in summary

### DISCUSSION THREAD

**Discussion by Lenbok** [magit/forge#592] [Support] [answered]
When I list pull requests in the magit status buffer, I see the PR number, title, and labels (if any). Is there a way to include the PR author (I saw some mention in issues about it being supported but I can't find out how to enable that), and ideally I'd also like to indicate if my review has been requested.

(I see it's possible to add whole extra status sections for authored pull requests and review requested, but really just want to augment the default list).

**Comment by tarsius** [comment, marked as answer]
In the status buffer type `L L` to show additional information in the margin. That should show the author and creation date. If not, then use the other margin commands, listed below the mentioned command, to change what is being displayed and how.

When you visit a pull-request, there is "Assignees" header in that buffer, but it isn't currently possible to show that in the default listing.

**Comment by Lenbok** [comment]
Wow, I just learned a new feature - I had no idea you could customize the status buffer like that! (I had thought the `L` prefix would be for doing something when going to the log view rather than the status buffer). For now I'll generally enable the margin details as you mentioned and use `(add-hook 'magit-status-sections-hook 'forge-insert-requested-reviews)` to have an extra section (since it only appears when relevant).

**Reply by tarsius** [reply]
`L` is for changing details about the logs *in the current buffer*. `D` does the same for diffs.

Originally the margin was only used to display additional information about logs, that is how it ended up in `L`. Once it grew beyond that, it made sense to also add it to `D`.

**Reply by Lenbok** [reply]
BTW @tarsius , is there a way that I can save this setting, so that whenever I open a status buffer it has the margin visible and details showing? I tried `w` (which sounded like it should save settings), but it didn't seem to save the margin settings.

**Reply by tarsius** [reply]
`w` safes the arguments presented in the transient buffer, which are usually consumed by passing them to suffix commands.

The commands that change margin settings work differently, they don't consume any arguments from the transient. (Note how there is no `--margin-style` argument in the transient.)

That doesn't mean that it might not make sense to allow saving these settings from the transient, but this comes with serious complications. I plan to go back to the drawing board on matters like this, sometime this winter, but until then, you will have to use the current method for customizing this:

Use `M-x customize-group RET magit-margin RET` to set the defaults.  Also see the "Log Margin" section in the manual.
